app-localize-behavior.html

```diff
@@ -52,6 +52,33 @@
 *        &lt;/script>
 *     </dom-module>
 *
+*
+* If the resources stored in your external file are for a single language and
+* so are not nested inside any language keys, you can pass an optional
+* `language` parameter to store the fetched resources inside that key.
+* This complements the optional third parameter, `merge`, nicely: If you pass
+* `merge = true`, the fetched resources will be merged into any existing
+* resources rather than clobbering them. Object.assign is used for the merge,
+* but you can also pass your own merge function to use that instead.
+* Finally, if you pass an optional fourth parameter, `preprocessor`, that
```

Comment:
Uhhh it feels like we're throwing a lot of stuff in. Why is `preprocessor` needed? Why is `merge` needed? Do you actually have examples when you would need this, or "just in case" ?

Reply:
The `preprocessor` param was an attempt to resolve #26 and #103 while I was at it. Since I currently only need #75 fixed, I'm happy to drop `preprocessor` from this PR and just did so in 2e60b42.